src/components/CountryTable.tsx

```diff
@@ -20,31 +22,77 @@ const CountryTable: React.FC<CountryTableProps> = ({ data }) => {
       </IonRow>
       <HeadRow />
       {data &&
-        Object.keys(data).map((key) => <ItemRow key={key} item={data[key]} />)}
+        Object.keys(data).map((key) => (
+          <ItemRow
+            style={
+              data[key] && data[key].CountryCode === myCoutryCode
+                ? "country-row"
+                : "item-row"
+            }
+            key={key}
+            item={data[key]}
+            defaultCountry={myCoutryCode}
+          />
+        ))}
     </IonGrid>
   );
 };
 
 const HeadRow: React.FC = () => (
   <IonRow className="head-row">
-    <IonCol>Location</IonCol>
+    <IonCol size="3">Location</IonCol>
     <IonCol>Confirmed</IonCol>
     <IonCol>Dead</IonCol>
     <IonCol>Recovered</IonCol>
+    <IonCol>C Today</IonCol>
+    <IonCol>D Today</IonCol>
+    <IonCol>R Today</IonCol>
   </IonRow>
 );
 
 type ItemRowProps = {
   item: ByCountryData;
+  defaultCountry: string;
+  style: string;
 };
 
-const ItemRow: React.FC<ItemRowProps> = ({ item }) => (
-  <IonRow className="item-row">
-    <IonCol>{item.Country}</IonCol>
-    <IonCol>{numberWithCommas(item.TotalConfirmed)}</IonCol>
-    <IonCol>{numberWithCommas(item.TotalDeaths)}</IonCol>
-    <IonCol>{numberWithCommas(item.TotalRecovered)}</IonCol>
-  </IonRow>
-);
+const ItemRow: React.FC<ItemRowProps> = ({ item, defaultCountry, style }) => {
+  return (
+    <IonRow className={style ? style : ""}>
```

Comment:
@tranthanhben2002 a bit misleading here. `style` prop is usually a style object, not string. Here you're passing to `className` prop of the root `<IonRow />` why don't name the passing prop `className` as well? External usage will be `<ItemRow className="abc" />`, perfectly looks like built-in components.